### PR TITLE
[Questionnaire] Order question

### PR DESCRIPTION
**Pull request by pitrackster**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no

Ordering question editor

### CODE REVIEW

**Comment by Pauooo** [comment]
Docker : http://pr-2083-1488891717.travis-build.claroline.net/claroline/

**Comment by LaurentGruber** [comment]
last docker : http://pr-2083-1488965894.travis-build.claroline.net/claroline/

File: plugin/exo/Entity/ItemType/OrderingQuestion.php [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+    /**
+     * @var string
+     */
+    const MODE_INSIDE = 'inside';
+
+    /**
+     * @var string
+     */
+    const MODE_BESIDE = 'beside';
```
Comment (Elorfin) [inline plugin/exo/Entity/ItemType/OrderingQuestion.php]: Can you document those constants please (the fourth) ?

File: plugin/exo/Entity/ItemType/OrderingQuestion.php [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+
+    /**
+     * Add item.
+     *
+     * @param OrderingItem $item
+     */
+    public function addItem(OrderingItem $item)
+    {
+        if (!$this->items->contains($item)) {
+            $item->SetQuestion($this);
```
Comment (Elorfin) [inline plugin/exo/Entity/ItemType/OrderingQuestion.php]: wrong case.

File: plugin/exo/Library/Item/Definition/OrderingDefinition.php [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+   */
+  public function __construct(
+      OrderingQuestionValidator $validator,
+      OrderingAnswerValidator $answerValidator,
+      OrderingQuestionSerializer $serializer)
+  {
+      $this->validator = $validator;
+      $this->answerValidator = $answerValidator;
+      $this->serializer = $serializer;
+  }
```
Comment (Elorfin) [inline plugin/exo/Library/Item/Definition/OrderingDefinition.php]: There is problem with indentation.

**Pull request merged**